spec.md

```diff
@@ -498,10 +531,6 @@ message VolumeInfo {
   // NFS share). If set, it MUST be non-zero.
   uint64 capacity_bytes = 1;
 
```

Comment:
Access mode should still be in the returned `VolumeInfo` no? That way SP can satisfy with `access_mode` equal to or better than what was asked for?